### PR TITLE
Reviews db seeding

### PR DESCRIPTION
**Pull request by dmoygit**
These are the main changes for this branch:
1) fakerData.js - This is creating fake data.  I did it in batches to make sure that the entries will show up chronologically (as they would if they were real)
2) seedDB.js - this uses the fake data, from FakerData and populates MySQL database.
3) package.json - added the script to populate the database ("seed").

### CODE REVIEW

File: database/fakerData.js [hunk 1]
```diff
@@ -1,11 +1,156 @@
...
+        experience_id: faker.random.number({min:1, max:100}),
+        username: faker.name.firstName(),
+        avatar: faker.image.avatar(),
+        date: '20-02-01',
+        stars: faker.random.number({min:0, max:5}),
+        review: faker.lorem.sentence()
+      };
+      results.push(user);
+    }
+    if (i >= 200 && i < 401) {
```
Comment (yerx) [inline database/fakerData.js]: The first if statement goes from 0 to 200. Would this if statement go from 201 to 400 so something like if i > 200 && i < 401?

File: database/fakerData.js [hunk 2]
```diff
@@ -1,11 +1,156 @@
...
+      };
+      results.push(user);
+    }
+
+  }
+
+  return results;
+}
+
+
```
Comment (yerx) [inline database/fakerData.js:156]: Nice implementation of using faker. I am taking notes 👍

**Review by yerx — APPROVED** [review]
Lgtm. I have one comment, but per your demo, it looks like the code is working.

File: database/fakerData.js (hunk 1 above)
Comment (dmoygit) [reply database/fakerData.js]: I have fixed this and it will be in my next commit

**Pull request merged**